src/lib/expression/abstract_expression.hpp

```diff
@@ -64,10 +64,20 @@ class AbstractExpression : public std::enable_shared_from_this<AbstractExpressio
 
   virtual std::shared_ptr<AbstractExpression> deep_copy() const = 0;
 
+  /**
+   * @return the expression's column name or, optionally, a more detailed description of the expression
+   */
+  enum class DescriptionMode {
+    ColumnName,  // returns only the column name (e.g., name)
```

Comment:
What's the meaning of the `e.g., name`?

Reply:
Mmmh, "name" is a bad example for a column name. :D We don't really need an example here, anyway.